src/engine/testAi.ts

```diff
@@ -1,11 +1,31 @@
 import { ChatCompletionRequestMessage } from 'openai';
 import { AiEngine } from './Engine';
+import { getConfig } from '../commands/config';
+
+export const TEST_MOCK_TYPES = [
```

Comment:
In the testing of OCO_PROMPT_MODULE=@commitlint, a TestAi that returns JSON data, not just a string, was needed. Therefore, I added a setting to select the output type of TestAi for testing purposes.